src/EditorFeatures/Core/Classification/Semantic/ClassificationUtilities.cs

```diff
@@ -196,12 +196,12 @@ public static SegmentedList<ITagSpan<IClassificationTag>> Convert(IClassificatio
             if (type == ClassificationType.Semantic)
             {
                 await classificationService.AddSemanticClassificationsAsync(
-                   document, snapshotSpan.Span.ToTextSpan(), options, classifiedSpans, cancellationToken).ConfigureAwait(false);
+                   document, new[] { snapshotSpan.Span.ToTextSpan() }, options, classifiedSpans, cancellationToken).ConfigureAwait(false);
```

Comment:
note: we'll want these to be immutable arrays for the final form.  but for the temp form, arrays are fine.